### DISCUSSION TITLE
[RIVER] How fix auto fan speed

### DISCUSSION THREAD

**Discussion by Suvero** [vwt12eh8/hassio-ecoflow#49] [General]
Hi ! Auto fan speed in my case (I using 3x River) is completely fail. Fan working only on „1” speed, but If I change auto fan speed to disabled, fan speed is „3”. Hot and quiet or cool and loud - bravo for china small devs. My question is - if so ecoflow did stupid alghoritm related to automatically fan control… where is „2” speed ? Medium ?

we can turn it on manually via local API ?

**Comment by vwt12eh8** [comment]
Auto Fan Speed works the same as the one in the official app.

When this is turned OFF, the fan always operates at the maximum speed (3).

When this is ON, I think that it operates at speed 2 if the internal temperature probably becomes a certain range, but a detailed condition is unknown.

**Comment by Suvero** [comment]
I just got answered from the support team. They told me that in auto mode, the fan only starts when 
1. the built-in inverter temperature is higher than 75 C degree, the battery temperature is higher than 45C degrees,
2. AC load power is over 300w

Maybe it will be useful for someone.
